contracts/HtsSystemContractJson.sol

```diff
@@ -62,6 +62,13 @@ contract HtsSystemContractJson is HtsSystemContract {
         }
 
         string memory json = mirrorNode().fetchTokenData(address(this));
+        if (vm.keyExistsJson(json, "_status")) {
```

Comment:
I was thinking here maybe we should set `tokenType = "NOTFOUND"`  or something similar. This would allows us to check whether an address is a token in methods such as `getTokenInfo` or `isToken` (yet to be implemented).